### ISSUE TITLE
[Feature Request] User Permissions on Comics (Directories etc.)

### ISSUE THREAD

**Issue by Leeingnyo** [hkalexling/Mango#13]
Hi!
Could you implement user permission on libraries (comics series or directories)?
I'm very appreciated to about multiple users feature.

but all users can read all comics now.
I want to open specific series for one user, hide other genre of comics.

It's just suggestion... you don't have to mind of this seriously.
I can't imagine myself this feature into UI...
In user setting page, it is shown up that the directory tree view with checkbox?
Hmm...

Thank you for reading!

**Comment by hkalexling** [comment]
Thanks for the suggestion! 

One way to archive what you want is

1. Allow admins to tag the series (e.g., add tags like `hero` or `hentai`)
2. Restrict non-admin users' access to series with specific tags only (e.g., user A can only see manga with tags `hero` or `action`)
3. Or we can blacklist a user from seeing manga with a certain tag (e.g., prevent user B from seeing manga with the tag `hentai`)

This is not a small feature, so I would like to spend some time and plan it carefully. Let me know what you think about the tagging solution I proposed.

**Comment by Leeingnyo** [comment]
It looks good!
It would be good that there is a method for tagging a same tag to hundreds of manga.

**Comment by hkalexling** [comment]
> It would be good that there is a method for tagging a same tag to hundreds of manga.

Yeah good point, and right now I can't think of a user-friendly way of doing this. I will think about it and leave this issue open.

**Comment by Leeingnyo** [comment]
Thank you!

**Comment by hkalexling** [comment]
For now you can work around this by running multiple instances of Mango on the same machine. Please see the [Wiki](https://github.com/hkalexling/Mango/wiki/Workarounds#multiple-libraries) for more details.

**Comment by Leeingnyo** [comment]
wow! good! thank you for your work and notice. I'll try on this
